README.md

```diff
@@ -15,109 +15,130 @@ You got to the repository that collect all style configurations used by DevExtre
+  - [`airbnb-typescript`](https://github.com/iamturns/eslint-config-airbnb-typescript)
   
 ## Spell Check
 
-Contains special words that should be ignored.
+Lists words that the spell check should ignore.
 
-- **How to use**\
-    &nbsp; &nbsp; Add to your *.eslintrc*: `"extends": ["devextreme/spell-check"]`
+- **Usage**
+    Add the following line to your *.eslintrc* file:
+    
+    ```javascript
+    "extends": ["devextreme/spell-check"]
+    ```
 
-- **Plugins**
+- **Required plugins**
   - [spellcheck](https://github.com/aotaduy/eslint-plugin-spellcheck)
 
 ## Renovation Declarations
 
-Use for the renovation declarations.
+- **Usage**    
+    Add the following line to your *.eslintrc* file:
+    
+    ```javascript
+    'extends': ['devextreme/renovation-declarations']
+    ```
+    
+- **Extended configurations**
+  - [`devextreme/typescript`](#typescript)
 
-- **How to use**\
-    &nbsp; &nbsp; Add to your *.eslintrc*: `'extends': ['devextreme/renovation-declarations']`
-- **Extends**
-  - `devextreme/typescript` [doc](#typescript)
+- **Overrides**
 
-- **Exceptions**
-  - Declarations used for generate target components in the development mode
+  - Allow importing devDependencies when in development mode:
 
-    ```javasxript
+    ```javascript
     'import/no-extraneous-dependencies': ['error', { devDependencies: true }]
     ```
 
-  - Declaration should contain two classes - component class and props class
+  - Allow two classes (component class and props class) in one file:
   
-    ```javasxript
+    ```javascript
     'max-classes-per-file': ['error', 2]
     ```
 
-  - Declaration is not a real React component and doesn't need it
+  - Do not require React. Declarations do not need it because they are not proper React components:
   
-    ```javasxript
+    ```javascript
     'react/react-in-jsx-scope': ['off']
     ```
 
 ## Jest Tests
 
-- **How to use**\
-    &nbsp; &nbsp; Add to your *.eslintrc*: `'extends': ['devextreme/jest']`
+- **Usage**    
+    Add the following line to your *.eslintrc* file:
+    
+    ```javascript
+    'extends': ['devextreme/jest']
+    ```
 
 - **Environment**
-  - `node` (Node.js global variables and Node.js scoping)
+  - `node` (Node.js global variables and scope)
   - `jest` (Jest global variables)
   
-- **Plugins**
+- **Required plugins**
   - [jest](https://github.com/jest-community/eslint-plugin-jest)
   - [jest-formatting](https://github.com/dangreenisrael/eslint-plugin-jest-formatting)
   
-- **Extends**
-  - `plugin:jest/recommended` [doc](https://github.com/jest-community/eslint-plugin-jest#recommended)
+- **Extended plugins**
+  - [`plugin:jest/recommended`](https://github.com/jest-community/eslint-plugin-jest#recommended)
   - `plugin:jest-formatting/recommended`
   
-- **Exceptions**
-  - Use devDependencies modules in the tests
+- **Overrides**
+
+  - Allow using devDependencies in the tests:
```

Comment:
Allow the use of devDependencies in tests:

please make this change to the other sentences below too